thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -51,7 +51,7 @@
 /**
  * This class operates an IoT device that is specified by {@link #onboard(String, String, String, JSONObject)} method.
  */
-public class ThingIFAPI implements Parcelable {
+public class ThingIFAPI<AliasType extends Alias> implements Parcelable {
```

Comment:
@yongpingchen 

This is cosmetic question. So this is not essential one.

In java, type parameter name is used to described by one uppercase letter like `T`.
Is `AliasType` good manner for Java?